### DISCUSSION TITLE
Fine tuning ML model - a couple of simple questions.

### DISCUSSION THREAD

**Discussion by connorourke** [PaddlePaddle/PaddleOCR#12427] [Q&A]
I would like to fine tune the ML model for detection (and eventually recognition) for an Arabic dataset I have. I would like to take some of the dataset and include them in the training. I have a couple of questions:

1) Following [the docs](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.3/doc/doc_en/detection_en.md) I understand the format the dataset should be in. Do the text boxes have to have separate words, or is it OK to have one box around e.g. entire sentence?
2) is having the text transcription in the dataset annotation important - is that information used during detection? I would have thought it was only for the recognition part, so I could leave this blank (I am not an Arabic speaker - so cant include the text transcription until a colleague sends it to me, which may be a while). 

Thanks!

**Comment by connorourke** [comment]
Also - can you tell me what model gets used when I set 

`PaddleOCR(lang='ar')`

It isnt entirely clear. Thanks

**Reply by Arslan-Mehmood1** [reply]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/51626734/67a51b65-2f45-4139-b784-011fd2b8f871)

by looking at the logs after initializing the paddleocr mode, we can see which models are loaded.
